### ISSUE TITLE
Wildcard Domain in KubernetesIngress get no rule in Router

### ISSUE THREAD

**Issue by tgruenert** [containous/traefik#6250]
Bug

Configured an wildcard domain (http) within Kubernetes Ingress. 
In Traefik1.7 it works as well as it should.
In Traefik2.1.3 there is no rule shown at dashboard.

### What did you expect to see?
In Traefik1.7 a wildcard domain was mapped into a regex route rule:
![image](https://user-images.githubusercontent.com/6465773/73269423-1854a600-41dd-11ea-8c18-33af0199abea.png)

### What did you see instead?
In Traefik2.1.3 this list keeps empty
![image](https://user-images.githubusercontent.com/6465773/73269480-2e626680-41dd-11ea-952a-d9d944e6f962.png)

But i see an entry under HTTP Services that say the right generated rule will be used by a router.
![image](https://user-images.githubusercontent.com/6465773/73270644-5b177d80-41df-11ea-85fa-842c17ebf1f3.png)

Please ignore the fact that the name includes "ssl". This is just the deployment name. Service and ingress are tested on port 80.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
2.1.3
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```yaml
...
 containers:
  - args:
    - --log=true
    - --log.level=INFO
    - --providers.kubernetesIngress.ingressEndpoint.ip=127.0.0.1
    - --entryPoints.http.address=:80
    - --entryPoints.https.address=:443
    - --providers.kubernetesingress=true
    - --providers.kubernetescrd=false
    - --api=true
    - --api.dashboard=true
    - --api.insecure=true
    - --ping=true
    - --log.level=DEBUG
    - --accesslog=true
```

ingress.yaml
```yaml
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    field.cattle.io/publicEndpoints: '[{"addresses":["127.0.0.1"],"port":80,"protocol":"HTTP","serviceName":"default:demo-ssl-demo-ssl-wildcard","ingressName":"default:demo-ssl-demo-ssl-wildcard","hostname":"*.somedomain.tld","path":"/","allNodes":true}]'
    kubernetes.io/ingress.class: traefik
    traefik.ingress.kubernetes.io/priority: "1"
  creationTimestamp: "2020-01-27T13:19:26Z"
  generation: 3
  labels:
    app.kubernetes.io/instance: demo-ssl
    app.kubernetes.io/managed-by: Tiller
    app.kubernetes.io/name: demo-ssl-wildcard
    app.kubernetes.io/version: "1.0"
    helm.sh/chart: demo-ssl-wildcard-0.1.0
  name: demo-ssl-demo-ssl-wildcard
  namespace: default
  resourceVersion: "56777724"
  selfLink: /apis/extensions/v1beta1/namespaces/default/ingresses/demo-ssl-demo-ssl-wildcard
  uid: 524e0a63-09cc-4886-a013-e387834680d0
spec:
  rules:
  - host: '*.somedomain.tld'
    http:
      paths:
      - backend:
          serviceName: demo-ssl-demo-ssl-wildcard
          servicePort: 80
        path: /
```

**Comment by tgruenert** [comment]
sorry - did´nt check the pagination at rules list. 
rule is shown up.

but did not work right now. calling http get status 404.

**Comment by dtomcej** [comment]
But the corresponding rule built was:

```
Host(`*.example.com`) && PathPrefix(`/`)
```

and not
```
HostRegexp(`{subdomain:[a-zA-Z0-9-]+}.example.com`) && PathPrefix(`/`)
```

which would render the wildcard unusable (since its an exact match, not a regex)

**Comment by tgruenert** [comment]
@dtomcej yes the first rule was build. (It is also show on third screenshot partly.)

**Comment by traefiker** [comment]
Closed by #6251.

**Comment by tgruenert** [comment]
@dtomcej Thank you for that incredible fast response and solution! Cheers ...
